ps-rule.yaml

```diff
@@ -0,0 +1,59 @@
+#
+# PSRule for Azure configuration
+#
+
+# Please see the documentation for all configuration options:
+# https://aka.ms/ps-rule-azure
+
+# Use rules from the following modules/
+include:
+  module:
+  - 'PSRule.Rules.Azure'
+
+# Require a minimum version of modules that include referenced baseline.
+requires:
+  PSRule.Rules.Azure: '@pre >=1.14.2'
+
+# Reference the repository in output.
+repository:
+  url: https://github.com/Azure/ALZ-Bicep
+
+execution:
+  # Ignore warnings for resources and objects that don't have any rules.
+  notProcessedWarning: false
+
+configuration:
+  # Enable expansion for Bicep source files.
+  AZURE_BICEP_FILE_EXPANSION: true
+
+  # Expand Bicep module from Azure parameter files.
+  AZURE_PARAMETER_FILE_EXPANSION: true
+
+  # Set timeout for expanding Bicep source files.
+  AZURE_BICEP_FILE_EXPANSION_TIMEOUT: 15
+
+input:
+  pathIgnore:
+  # Ignore common files that don't need analysis.
+  - '**/bicepconfig.json'
+  - '*.md'
+  - '*.png'
+  - '.github/'
+
+  # Exclude Bicep docs files
+  - docs/scripts/callModuleFromACR.example.bicep
+
+  # Exclude Bicep module files
+  - 'infra-as-code/bicep/modules/**/*.bicep'
+  - 'infra-as-code/bicep/CRML/**/*.bicep'
+
+  # Include test files from modules
+  - '!infra-as-code/bicep/modules/**/*.tests.bicep'
```

Comment:
Is this in the correct part of the config file?

Or is this saying "don't ignore" because of the `!`?

Reply:
@jtracey93 Correct. It follows the git path spec. Entries are normally ignored. Entries beginning with `!` are re-included if previously ignored.

This is the standard example, but I can remove it until we work through where the snippets will be located.

Reply:
Awesome thanks @BernieWhite.

Can we tweak it to look for `.bicep` files in every module folder called `samples` and then inside this folder we will create the new "samples/tests" that PSRule can scan.

We can then get this merged into the `ps-rule` branch and then start adding the samples in and making the adjustments required and also solves another item for including samples bicep files as well 👍